tests/suites/paginatedTable/paginatedTable.ts

```diff
@@ -0,0 +1,126 @@
+import type {Locator, Page} from '@playwright/test';
+
+export class PaginatedTable {
+    private page: Page;
+    private tableSelector: Locator;
+    private searchInput: Locator;
+    private radioButtons: Locator;
+    private countLabel: Locator;
+    private tableRows: Locator;
+    private refreshButton: Locator;
+    private refreshIntervalSelect: Locator;
+    private headCells: Locator;
+
+    constructor(page: Page) {
+        this.page = page;
+        this.tableSelector = page.locator('.ydb-table-with-controls-layout');
+        this.searchInput = this.tableSelector.locator('.ydb-search input');
+        this.radioButtons = this.tableSelector.locator('.g-radio-button');
+        this.countLabel = this.tableSelector.locator('.ydb-entities-count .g-label__content');
+        this.headCells = this.tableSelector.locator('.ydb-paginated-table__head-cell');
+        this.tableRows = this.tableSelector.locator('.ydb-paginated-table__row');
+        this.refreshButton = page.locator('.auto-refresh-control button[aria-label="Refresh"]');
+        this.refreshIntervalSelect = page.locator('.cluster__auto-refresh-select');
```

Comment:
What do you think about adding `data-qa` attributes to elements instead of css class selectors?

Reply:
I think we don't really need it for now